macros/src/types/enum.rs

```diff
@@ -95,12 +95,26 @@ fn format_variant(
         },
         Tagged::Adjacently { tag, content } => match &variant.fields {
             Fields::Unnamed(unnamed) if unnamed.unnamed.len() == 1 => {
-                let FieldAttr { type_override, .. } =
-                    FieldAttr::from_attrs(&unnamed.unnamed[0].attrs)?;
+                let FieldAttr {
+                    type_as,
+                    type_override,
+                    ..
+                } = FieldAttr::from_attrs(&unnamed.unnamed[0].attrs)?;
+
+                if let (Some(_type_as), Some(_type_override)) = (&type_as, &type_override) {
```

Comment:
Since you're just checking if the `Options` are `Some`, the following code is easier to understand:
```rs
if type_as.is_some() && type_override.is_some() {
```
It removes the need for `if let`, tuples and borrowing